frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -452,11 +466,10 @@ export const DetachedTestResultsList = ({
                   setStartDateError("");
                   setEndDateError("");
                 }}
-                disabled={
```

Comment:
⚠️ updated this to address a code smell related to code complexity being over 15